reconcile/utils/terrascript_client.py

```diff
@@ -3155,6 +3175,154 @@ def is_elasticsearch_domain_name_valid(name):
         pattern = r'^[a-z][a-z0-9-]+$'
         return re.search(pattern, name)
 
+    @staticmethod
+    def elasticsearch_log_group_identifier(
+        domain_identifier: str,
+        log_type: ElasticSearchLogGroupType
+    ) -> str:
+        log_type_name = log_type.value.lower()
+        return f'OpenSearchService__{domain_identifier}__{log_type_name}'
+
+    def _elasticsearch_aggregate_log_groups_per_account(
+        self
+    ) -> list[ElasticSearchLogGroupInfo]:
+        log_group_infos = []
+        for resources in self.account_resources.values():
+            for i in resources:
+                res = i['resource']
+                ns = i['namespace_info']
+                if not ns.get('managedTerraformResources'):
+                    continue
```

Comment:
do you need the check here again? i think it is check already earlier in the program flow [here](https://github.com/app-sre/qontract-reconcile/blob/b066ce7373248e991029378c29a0f560c662be08/reconcile/terraform_resources.py#L446)

Reply:
Great catch!!

Removed the redundant check :+1: